tiff-2-png-converter.py

```diff
@@ -2,348 +2,282 @@
+            equilize_histogram_function = iph.equalize_image_histogram_custom
+        else:
+            raise ValueError("ERROR: equilize_histogram value is not valid")
+
+    list_files, width, height = list_folder_files(src)
+
+    seqencer = Sequence(list_files)
+    
+    sequence_format = seqencer.format("%4l %r").split()
+    ## parser.format returns a separated by space string describing the properties of the found sequence
+    ## [0] sequence length
+    ## [1] implied range, start-end  
+
+    num_files_in_sequence = int(sequence_format[0])
+    if num_files_in_sequence == 0:
+        logging.error("No sequence found in source folder")
+        raise ValueError("No sequence found")
+    
+    confirm_txt = "Found a sequence of "+str(num_files_in_sequence) + " images "+ sequence_format[1] + ". Do you want to proceed?"
+
+    max_pixel = 0
+    min_pixel = 0
+    histogram = np.zeros((width * height *num_files_in_sequence ),dtype=np.uint16)
+    if typer.confirm(confirm_txt, abort=False): 
+        images_in_sequence = [None] * num_files_in_sequence
+        bits_per_sample = ""
+        with typer.progressbar(
+            range(num_files_in_sequence), label="Reading images from directory"
+        ) as progress:
+            for slice in progress:
+                file_full_path = list_files[slice]
+                if seqencer.contains(file_full_path):
+                    img = tiff.imread(list_files[slice])
+                    max_pixel = (max_pixel, np.max(img))[np.max(img) > max_pixel]
+                    min_pixel = (min_pixel, np.min(img))[np.min(img) < min_pixel]
+                    ## check all images have the same data type
+                    if bits_per_sample == "":
+                        bits_per_sample = img.dtype
+                    elif bits_per_sample != img.dtype:
+                        raise ValueError(
+                        "image "
+                        + file_full_path
+                        + "has a different sample type "
+                        + bits_per_sample
+                        + " expected: "
+                        + bits_per_sample
+                        )
+                    if img.ndim > 2:
+                        images_in_sequence[slice] = img[channel]
+                    else:
+                        images_in_sequence[slice] = img
+        
+        
+        logging.info("MAX Value in volume: "+str(max_pixel))
+        logging.info("MIN Value in volume: "+str(min_pixel))
+        
+
+        #convert to 8 bit
+        images_in_sequence_8_bit = [None] * num_files_in_sequence
+        with typer.progressbar(
+            range(num_files_in_sequence), label="Rescaling to 8 bit signal"
+            ) as progress:
+                for slice in progress:
+                    _img = images_in_sequence[slice]
+                    img_8_bit = np.subtract(_img,min_pixel)
+                    img_8_bit = np.true_divide(_img,np.subtract(max_pixel,min_pixel))
```

Comment:
I'm curious why you use `np.subtract(max_pixel,min_pixel)` instead of `max_pixel-min_pixel` and `np.multiply(img_8_bit,255)` instead of `255* img_8_bit`?